src/styles/layout/navigation.less

```diff
@@ -0,0 +1,1333 @@
+/* -----------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+sidebar.less
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+----------------------------------------------------------------------------- */
+
+
+
+.navigation {
+
+  position: fixed;
+  left: -100%;
+  width: 100%;
+
+  ul.sidebar-menu {
+
+    .clearfix();
+    .shape-style-variant(sidebar-menu);
+    margin-bottom: 0px;
+
+    li {
+
+      display: block;
+      .element-text-size(sidebar-menu-item, null);
+      .element-spacing(sidebar-menu-item, null, null);
+      .text-style-variant(sidebar-menu-item);
+      .shape-style-variant(sidebar-menu-item);
+
+      &:first-child {
+
+        margin-top: 0px;
+
+      }
+
+      .icon {
+
+        display: none;
+        
+      }
+
+      a {
+
+        display: block;
+        text-decoration: none !important;
+        .element-spacing(sidebar-menu-item-link, null, null);
+        .text-style-variant(sidebar-menu-item-link);
+        .shape-style-variant(sidebar-menu-item-link);
+
+        &:hover {
+
+          .text-style-variant(sidebar-menu-item-link-hover);
+          .shape-style-variant(sidebar-menu-item-link-hover);
```

Comment:
I think a commit wasn't pushed. In multiple locations in this file, `.shape` comes after `.text`.